ci/acceptance-test/tests/cmd_log_test.go

```diff
@@ -0,0 +1,34 @@
+// +build acceptance
+
+package tests
+
+import (
+	"os/exec"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/tidwall/gjson"
+)
+
+func TestOpenVDCLog(t *testing.T) {
+	stdout, _ := RunCmdAndReportFail(t, "openvdc", "run", "centos/7/lxc")
+	instance_id := strings.TrimSpace(stdout.String())
+	defer RunCmd("openvdc", "destroy", instance_id)
+
+	waitUntil(t, 5 * time.Minute, func()error {
+		cmd := exec.Command("openvdc", "show", instance_id)
+		buf, err := cmd.CombinedOutput()
+		if err !=nil {
+			return err
+		}
+		result := gjson.GetBytes(buf, "instance.lastState.state")
+		if result.String() == "RUNNING" {
+			return nil
+		}
+		time.Sleep(5 * time.Second)
+		return WaitContinue
+	})
```

Comment:
Since very similar code from the console service branch has been merged, now would be a good time to merge that in here and refactor it to a function so we can re-use it. https://github.com/axsh/openvdc/blob/master/ci/acceptance-test/tests/cmd_console_test.go#L18-L30